packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx

```diff
@@ -444,6 +454,7 @@ export const CommentTree = ({
                   </div>
                 )}
                 <CommentCard
+                  scrollToElement={scrollToElement}
```

Comment:
let's use existing callbacks, we can remove this prop and use the `onReply` callback below to trigger this `scrollToElement()` action.

Reply:
hm, it should still work since onReply isn't aync, going to try.